docs/apache-airflow/howto/custom-view-plugin.rst

```diff
@@ -24,12 +24,12 @@ core UI using the Plugin manager
 
 This is an example plugin for Airflow that displays absolutely nothing.
 
-In this plugin, two object reference are derived from the base class
+In this plugin, two objects reference are derived from the base class
 ``airflow.plugins_manager.AirflowPlugin``. They are flask_blueprints and
 appbuilder_views
 
 Using flask_blueprints in Airflow plugin, the core application can be extended
-to support the application that is customized to view Empty Plugin.
+to support the customized application to view the Empty Plugin.
```

Comment:
I think the _the_ is less right. Empty Plugin is a proper noun here and does not need a definite article.

Reply:
sure